webui/react/src/pages/TrialLogs.tsx

```diff
@@ -91,6 +91,7 @@ const TrialLogs: React.FC = () => {
     <Page hideTitle title={title}>
       <LogViewer
         disableLevel
+        disableLineNumber
```

Comment:
question: do we want to also disable it for master logs? especially since the user can never go back to the beginning of logs since we don't keep all the logs around?

Reply:
discussed with @vishnu2kmohan. trial logs are going to be serving local ids soon, the line numbers in master do not get copied for clipboard interactions and they serve as a decent marker to know where in the logs you are looking at. Decided to keep them for now.